pentatrion/upload-bundle/0.2/config/packages/pentatrion_upload.yaml

```diff
@@ -0,0 +1,7 @@
+pentatrion_upload:
+    # default configuration
```

Comment:
This comment is not needed. 

If you want to, add a link to your documentation.